org.tintero.Tintero.json

```diff
@@ -0,0 +1,46 @@
+{
+  "app-id": "org.tintero.Tintero",
```

Comment:
there is no obvious link between `tintero.org` and a piece of software. Is this domain really under the control of this application developers?